lib/battle_boats/random_fleet_deployer.rb

```diff
@@ -0,0 +1,30 @@
+require_relative "board"
+require_relative "coordinate"
+
+module BattleBoats
+  class RandomFleetDeployer
+    def initialize(board: BattleBoats::Board.new(state: :enemy))
+      @board = board
+    end
+
+    def deploy(fleet)
+      fleet.ships.each do |ship|
+        coin_flip = ["heads", "tails"].sample
```

Comment:
IS there a way to do this randomization without "magic" strings

Reply:
Absolutely!